doctoshotgun.py

```diff
@@ -653,6 +663,8 @@ def main(self, cli_args=None):
                             action='append', help='exclude centers')
         parser.add_argument('--center-exclude-regex',
                             action='append', help='exclude centers by regex')
+        parser.add_argument('--additional-center', '-ac',
```

Comment:
Until now, we only had single-letter abbreviations for arguments but it looks like we are running out of usable letters. Still it should be a deliberate decision to extend to two-letter abbreviations. @rbignon what do you think?